doc/articles/uwp-vs-winui3.md

```diff
@@ -1,11 +1,11 @@
 # WinUI 3 and Uno Platform
 
-WinUI 3 is the [next generation of Microsoft's Windows UI library](https://docs.microsoft.com/en-us/windows/apps/winui/). Scheduled for release in 2021, it succeeds the UWP XAML framework as Microsoft's actively-developed native UI platform. WinUI 3 supports Windows Desktop apps through [Project Reunion](https://docs.microsoft.com/en-us/windows/apps/project-reunion/).
+WinUI 3 is the [next generation of Microsoft's Windows UI library](https://docs.microsoft.com/en-us/windows/apps/winui/). It succeeds the UWP XAML framework as Microsoft's actively-developed native UI platform for Windows. WinUI 3 supports Windows Desktop apps through [Windows AppSDK](https://docs.microsoft.com/en-us/windows/apps/project-reunion/).
```

Comment:
nit: the "project-reunion" url now points to the "windows-app-sdk" url. Might as well use the new url in-case the re-route is ever removed in the future

```suggestion
WinUI 3 is the [next generation of Microsoft's Windows UI library](https://docs.microsoft.com/en-us/windows/apps/winui/). It succeeds the UWP XAML framework as Microsoft's actively-developed native UI platform for Windows. WinUI 3 supports Windows Desktop apps through [Windows AppSDK](https://docs.microsoft.com/en-us/windows/apps/windows-app-sdk/).
```